### ISSUE TITLE
Live Demo Error

### ISSUE THREAD

**Issue by bradley-holt** [ibm-watson-data-lab/shopping-list-react-pouchdb#54]
The live demo hosted on GitHub Pages loads a blank screen. The console displays the following error:

```
Uncaught TypeError: Cannot read property 'applitude' of undefined
    at Object.<anonymous> (main.5a4ae57a.js:1)
    at t (main.5a4ae57a.js:1)
    at Object.<anonymous> (main.5a4ae57a.js:1)
    at t (main.5a4ae57a.js:1)
    at Object.<anonymous> (main.5a4ae57a.js:1)
    at t (main.5a4ae57a.js:1)
    at main.5a4ae57a.js:1
    at Object.<anonymous> (main.5a4ae57a.js:1)
    at t (main.5a4ae57a.js:1)
    at Object.<anonymous> (main.5a4ae57a.js:1)
```

**Comment by isc-rsingh** [comment]
I get 
```
index.js:28 Uncaught TypeError: c.ShoppingListFactory is not a constructor
    at Object.<anonymous> (index.js:28)
    at t (bootstrap 9f1683f471ddd86066f8:19)
    at Object.<anonymous> (main.30831e87.js:75069)
    at t (bootstrap 9f1683f471ddd86066f8:19)
    at bootstrap 9f1683f471ddd86066f8:65
    at bootstrap 9f1683f471ddd86066f8:65
```

@vabarbosa I think this is still part of the React build issue #28 and needs a holistic fix. I'm re-opening that one.

**Comment by vabarbosa** [comment]
@bradley-holt this should hopefully be resolved with an updated `shopping-list-model-js`
see: https://github.com/ibm-watson-data-lab/shopping-list-model-js/pull/20

**Comment by vabarbosa** [comment]
https://ibm-watson-data-lab.github.io/shopping-list-react-pouchdb/
resolved with fix to #28 (PR #59)

**Comment by bradley-holt** [comment]
LGTM.
